dc/core/ContentWindow.cpp

```diff
@@ -110,6 +110,16 @@ void ContentWindow::setContent( ContentPtr content )
     createInteractionDelegate();
 }
 
+ContentWindowController* ContentWindow::getController()
```

Comment:
Yes, I can add checks in the InteractionDelegates to be coherent, although we should never experience such situations in the contexts where they are used. I can also improve the unit test a bit.